server/plugin_test.go

```diff
@@ -38,8 +38,17 @@ func TestPlugin(t *testing.T) {
 func TestSpecialCases(t *testing.T) {
 	links := make([]*Link, 0)
 	links = append(links, &Link{
+		Pattern:  "https://mattermost.com",
+		Template: "[the mattermost portal](https://mattermost.com)",
+	}, &Link{
 		Pattern:  "(Mattermost)",
 		Template: "[Mattermost](https://mattermost.com)",
+	}, &Link{
+		Pattern:  "https://mattermost.atlassian.net/browse/MM-(?P<jira_id>\\d+)",
```

Comment:
Is this materially different than line 56?

Reply:
Yes, there is no different, I think I fixed that in other PR.

Reply:
nevermind, I going to remove that test.